src/fsharp/vs/Symbols.fs

```diff
@@ -1100,7 +1100,8 @@ and FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
                 let rty = m.GetFSharpReturnTy(cenv.amap,range0,m.FormalMethodInst)
                 let argtysl = m.GetParamTypes(cenv.amap,range0,m.FormalMethodInst) 
                 mkIteratedFunTy (List.map (mkTupledTy cenv.g) argtysl) rty
-            | V v -> v.TauType
+            | V v -> let _, typ, _ = PrettyTypes.PrettifyTypes1 cenv.g v.TauType
```

Comment:
The need to prettify types can arise not only for functions and values but also for members, properties and events.  You should probably do those here too